lib/livebook_web/live/session_live/app_settings_component.ex

```diff
@@ -0,0 +1,156 @@
+defmodule LivebookWeb.SessionLive.AppSettingsComponent do
+  use LivebookWeb, :live_component
+
+  alias Livebook.Notebook.AppSettings
+
+  @impl true
+  def update(assigns, socket) do
+    changeset =
+      case socket.assigns do
+        %{changeset: changeset} when changeset.data == assigns.settings -> changeset
+        _ -> AppSettings.change(assigns.settings)
+      end
+
+    {:ok,
+     socket
+     |> assign(assigns)
+     |> assign(changeset: changeset)}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="p-6 max-w-4xl flex flex-col space-y-8">
+      <h3 class="text-2xl font-semibold text-gray-800">
+        App settings
+      </h3>
+      <.form :let={f} for={@changeset} phx-change="validate" phx-target={@myself} autocomplete="off">
+        <div class="flex flex-col space-y-4">
+          <.text_field field={f[:slug]} label="Slug" spellcheck="false" phx-debounce />
+          <div class="flex flex-col space-y-1">
+            <.checkbox_field
+              field={f[:access_type]}
+              label="Password-protected"
+              checked_value="protected"
+              unchecked_value="public"
+            />
+            <%= if Ecto.Changeset.get_field(@changeset, :access_type) == :protected do %>
+              <.password_field field={f[:password]} spellcheck="false" phx-debounce />
+            <% end %>
+          </div>
+          <.radio_button_group_field
+            field={f[:multi_session]}
+            options={[{"false", "Single"}, {"true", "Multi"}]}
```

Comment:
There is a lot of space available, I think we can fill it in better and help the "Multi-session" and "Single-session" more prominent?

```suggestion
            options={[{"false", "Single-session"}, {"true", "Multi-session"}]}
```